src/uct/ib/base/ib_md.c

```diff
@@ -1713,6 +1708,33 @@ void uct_ib_md_close(uct_md_h uct_md)
     ucs_free(md);
 }
 
+ucs_status_t uct_ib_verbs_md_open_common(struct ibv_device *ibv_device,
+                                         const uct_ib_md_config_t *md_config,
+                                         uct_ib_md_t *md)
+{
+    ucs_status_t status;
+    int num_mrs;
+
+    md->config = md_config->ext;
+
+    status = uct_ib_device_query(&md->dev, ibv_device);
+    if (status != UCS_OK) {
+        return status;
+    }
+
+    uct_ib_md_parse_relaxed_order(md, md_config);
+    num_mrs = 1;    /* UCT_IB_MR_DEFAULT */
+
+    if (md->relaxed_order) {
+        ++num_mrs;  /* UCT_IB_MR_STRICT_ORDER */
+    }
```

Comment:
maybe we can push this code into uct_ib_md_open_common?

Reply:
so that we can remove the duplicate code in uct_ib_mlx5_parse_relaxed_order?